java/arcs/core/allocator/Allocator.kt

```diff
@@ -191,4 +231,41 @@ class Allocator(val hostRegistry: HostRegistry) {
         hostRegistry.availableArcHosts()
             .firstOrNull { host -> host.isHostForParticle(particle) }
             ?: throw ParticleNotFoundException(particle)
+
+    companion object {
+        /** Schema for persistent storage of [PlanPartition] information */
+        private val SCHEMA = Schema(
+            listOf(SchemaName("partition")),
+            SchemaFields(
+                mapOf(
+                    "arc" to FieldType.Text,
+                    "host" to FieldType.Text
+                ),
+                mapOf(
+                    "particles" to FieldType.Text
+                )),
+            "whyIsThisHere?"
```

Comment:
I know you don't approve of the schema hash idea, but could you pick something more intelligible (and more obviously unique) here? Ideally you could do `SCHEMA.hashCode()` -- but you can't get the hash of an object you haven't created yet... Using a constant like "plan-partition-schema-hash" would be fine.

Reply:
I put that there as a reminder to find out.. I think I've managed to convince Jason it isn't necessary BTW; we'll use the Schema itself instead of a hash post-January.

Reply:
(-hash would be a lie)